test/dummy/app/tasks/maintenance/params_task.rb

```diff
@@ -20,6 +20,10 @@ class ParamsTask < MaintenanceTasks::Task
     attribute :enum_attr, :integer
 
     validates_inclusion_of :enum_attr, in: [100, 200, 300], allow_nil: true
+    validates_inclusion_of :enum_attr, within: [100, 200, 300], allow_nil: true
+    validates_inclusion_of :enum_attr, in: proc { [100, 200, 300] }, allow_nil: true
+    validates_inclusion_of :enum_attr, in: ->(_record) { [100, 200, 300] }, allow_nil: true
+    validates_inclusion_of :enum_attr, in: :enum_array, allow_nil: true
```

Comment:
These are not super helpful as they don't really do anything. They're ignored in the view because we only look for the first `:inclusion` validator (which I think is OK overall).

Reply:
Yeah, maybe when adding a test case to ensure that procs / symbols don't blow up, we can add a validator on another attribute?